packages/usa-in-page-navigation/src/test/in-page-navigation.spec.js

```diff
@@ -175,6 +180,40 @@ tests.forEach(({ name, selector: containerSelector }) => {
       assert(window.scroll.calledOnceWith(sinon.match({ top: 880 })));
     });
 
+    describe("Minimum heading count tests", () => {
+      beforeEach(() => {
+        body.innerHTML = TEMPLATE;
+        theNav = document.querySelector(THE_NAV);
+      });
+
+      afterEach(() => {
+        body.innerHTML = "";
+        window.location.hash = "";
+      });
```

Comment:
**issue**:  The fact that we reset in `beforeEach()` and `afterEach()` makes me lean towards separating this test into its own file.

Reply:
This was fixed in [1d137b2](https://github.com/uswds/uswds/pull/6205/commits/1d137b2ac71a01132b6e9a5fedb92b6ac936895f) and further updated since then